utils/api/benefits/oasBenefit.ts

```diff
@@ -251,6 +251,7 @@ export class OasBenefit extends BaseBenefit<EntitlementResultOas> {
    * The yearly amount of "clawback" aka "repayment tax" the client will have to repay.
    */
   private get clawbackAmount(): number {
+    if (!this.input.income.client) return 0
```

Comment:
can we merge this condition with the following one using ||

Reply:
good point Yan, checking in the change